src/components/genre-select/genre-select.tsx

```diff
@@ -72,19 +75,27 @@ export default function GenreSelect({
             }}
           >
             <div className="genre-select-list">
-              {genres.map((item) => (
-                <Option
-                  key={item.id}
-                  checked={value.includes(item.name)}
-                  onChange={handleOptionChange(item.name)}
-                >
-                  {item.name}
-                </Option>
-              ))}
+              {genres.map((item) => {
+                const currentGenre = item.name
+                  .split(' ')
+                  .map(
+                    (word) =>
+                      word.charAt(0).toUpperCase() + word.slice(1).toLowerCase()
```

Comment:
Can't it be done with text-transform: capitalize?

Reply:
Changed it to check genre option using the .some method.